server/routes/api/article.js

```diff
@@ -11,8 +12,9 @@ const router = express.Router();
 router.post('/articles', checkToken, upload.single('imgUrl'), ArticleValidations.validateCreateArticle, ArticleController.createArticle);
 router.get('/articles/feed', ArticleController.getAllArticles);
 router.get('/articles', checkToken, ArticleController.getUserArticles);
-router.get('/articles/:slug', checkToken, ArticleController.getSingleArticle);
+router.get('/articles/:slug', ArticleController.getSingleArticle);
```

Comment:
I removed the checkToken middleware to enable unauthenticated users to view an article.